cluster/aws/util.sh

```diff
@@ -1211,7 +1211,13 @@ function wait-master() {
   echo "  up."
   echo
 
-  until $(curl --insecure --user ${KUBE_USER}:${KUBE_PASSWORD} --max-time 5 \
+  if [[ $(curl -V | awk '{print $2}' | head -1) > 7.34.0 ]]; then
+    TLS_OPTS="--insecure --tlsv1.2"
```

Comment:
It seems like none of these are useful because we are always using --insecure. What extra security does this provide?